Makefile

```diff
@@ -8,8 +8,10 @@ CPPFLAGS = -Isrc -DSCALE -Iext/scale2x -Iext/psmplug -Iext/miniz
 
 # Network support
 CXXFLAGS += -DUSE_SOCKETS
-# Needed under Windows
-#LIBS += -lws2_32
+ifeq ($(OS),Windows_NT)
```

Comment:
This is GNU make syntax and therefore not really portable.
The basic Makefile should be compatible with BSD make at least.

Reply:
I don't know how to add BSD make support, so I've inquired about it.